File: mycuration/src/main/java/com/phicdy/mycuration/presentation/view/FilterListView.java
package com.phicdy.mycuration.presentation.view;

import android.support.annotation.NonNull;

import com.phicdy.mycuration.filter.Filter;

import java.util.ArrayList;

public interface FilterListView {
    void remove(int position);
    void notifyListChanged();
    void startEditActivity(int filterId);
    void initList(@NonNull ArrayList<Filter> filters);
}

File: mycuration/src/main/java/com/phicdy/mycuration/presentation/view/FilterListView.kt
package com.phicdy.mycuration.presentation.view

import com.phicdy.mycuration.data.filter.Filter

import java.util.ArrayList

interface FilterListView {
    fun remove(position: Int)
    fun notifyListChanged()
    fun startEditActivity(filterId: Int)
    fun initList(filters: ArrayList<Filter>)
}
